core/templates/components/concept-card/concept-card.directive.ts

```diff
@@ -78,6 +78,7 @@ angular.module('oppia').directive('conceptCard', [
               if (ctrl.currentConceptCard.getWorkedExamples().length > 0) {
                 ctrl.numberOfWorkedExamplesShown = 1;
               }
+              $rootScope.$apply();
```

Comment:
Add a TODO to remove this when this directive is migrated to angular.

Reply:
Done!